### PR TITLE
V2

### PR DESCRIPTION
**Pull request by TomFrink**
I have added some layout changes to Slack area and Bottom Nav for breakpoints on mobile. 
Added a slight background to containers for readability 
Added FontAwesome package and Icons, where needed.
Added a Static Value in Views.py for development 
I'm not 100% convinced I like the width 100% (see what I did there) but couldn't find a better way yet to maintain width @ different breakpoints.

### CODE REVIEW

**Review by geowa4 — CHANGES_REQUESTED** [review]
What's that `pip` file I see in this PR?

File: web/about/views.py [hunk 1]
```diff
@@ -38,8 +38,12 @@ def index(request):
...
         'has_registered': False,
         'chat_form': None,
-        'num_members': len(members()),
-        'num_active': len(active_members()),
+        # Prod with access to Slack API Token
+        # 'num_members': len(members()),
+        # 'num_active': len(active_members()),
+        # Dev Mode with no access to Slack API Token
+        'num_members': 900,
+        'num_active': 0,
```
Comment (geowa4) [inline web/about/views.py]: We'll need to put this back before merging. I'll see about making a patch you can integrate that handles the case of not having a valid token.

Comment (TomFrink) [reply web/about/views.py]: Got it. 
I'll adjust or remove that piece.

File: Pipfile [hunk 2]
```diff
@@ -17,6 +17,7 @@ django = "*"
 django-bootstrap4 = "*"
 requests = "*"
 django-ical = "*"
+django-fontawesome-5 = "*"
```
Comment (TomFrink) [inline Pipfile]: Changes to Pipfile to add FontAwesome for Icons.

File: web/about/views.py [hunk 3]
```diff
@@ -38,8 +38,12 @@ def index(request):
         'next_event': events.get_next(),
         'has_registered': False,
         'chat_form': None,
+        # Prod with access to Slack API Token
         'num_members': len(members()),
         'num_active': len(active_members()),
+        # Dev Mode with no access to Slack API Token
+        # 'num_members': 900,
+        # 'num_active': 0,
```
Comment (TomFrink) [inline web/about/views.py:46]: I commented out the toggle I was using in Dev.

File: web/about/views.py (hunk 1 above)
Comment (TomFrink) [reply web/about/views.py]: I commented out the other way. 
That way it's still there if someone needs it but can easily be removed.

**Pull request closed without merging**